sdk/ml/azure-ai-ml/azure/ai/ml/_azure_environments.py

```diff
@@ -4,8 +4,10 @@
 
```

Comment:
Please add test to validate this change. It can help to prevent any breaking changes to this logic. And appreciate the effort in putting this PR together. Thanks for the contribution